.packit.yaml

```diff
@@ -18,7 +18,7 @@ downstream_package_name: foreman
 
 actions:
   post-upstream-clone:
-    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/foreman/foreman/foreman.spec -O foreman.spec"
```

Comment:
we should not forget to drop the `[HACK] use evgeni's temporary repo with webpack5 builds` commit before merging (we can also do so now, the only downside would be failing packit builds, but that's IMHO fine)